schema-registry/src/test/scala/net/manub/embeddedkafka/schemaregistry/EmbeddedKafkaWithSchemaRegistryTraitSpec.scala

```diff
@@ -0,0 +1,49 @@
+package net.manub.embeddedkafka.schemaregistry
+
+import net.manub.embeddedkafka.EmbeddedKafkaSpecSupport
+
+class EmbeddedKafkaWithSchemaRegistryTraitSpec
```

Comment:
Could we drop the "WithSchemaRegistry" part since this class is already on a different package?